完善骨骼动画相关 (#1553)
cocos-creator/creator-docs: en/editor/components/dragonbones.md

![dragonbones](./dragonbones/properties.png)

In the **Hierarchy** panel, select the node where you want to add the ArmatureDisplay component, and then click the **Add Components -> DragonBones -> ArmatureDisplay** button under the **Inspector** panel to add the ArmatureDisplay component to the node.

> - For the use of the ArmatureDisplay component, please refer to the [DragonBones example case](https://github.com/cocos-creator/test-cases-3d/tree/v3.0/assets/cases/dragonbones) for details.

Reviewer: ```suggestion
- For information on using the ArmatureDisplay component, please refer to the [DragonBones example case](https://github.com/cocos-creator/test-cases-3d/tree/v3.0/assets/cases/dragonbones) for details.
```
